docs/build/html/_static/language_data.js

```
@@ -0,0 +1,192 @@
+    re = /^(.+?)eed$/;
+    re2 = /^(.+?)(ed|ing)$/;
+    if (re.test(w)) {
+      var fp = re.exec(w);
+      re = new RegExp(mgr0);
+      if (re.test(fp[1])) {
+        re = /.$/;
+        w = w.replace(re,"");
+      }
+    }
+    else if (re2.test(w)) {
+      var fp = re2.exec(w);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'fp'. Consider to delete it or rename it.

'fp' is defined here:


(lint/suspicious/noRedeclare)